Document target-scope setting in JSON API (#861)
digital-asset/docs.daml.com: docs/2.10.0/docs/json-api/index.rst

The HTTP JSON API Service does not validate the token but may need to decode it to extract information that can be used
to fill in request fields for party-specific request. How this happens depends partially on the token format you are using.

If the Canton participant to which the JSON API is connected uses target-scopes, the configuration has to reflect that

> through the following setting:

Author: I can see how confusing my original words have been. What I meant is that these new settings apply to the JSON API component which in 2.x release line is completely separate process. So if participant has target-scopes in its configuration, also the JSON-API needs them in its configuration.
Can you help me find the right words @carrielaben-da ?

> through the following setting:

Author: perfect!

> through the following setting:

Author: To connect to a Canton participant that uses target-scopes, the JSON API must also include target-scopes in its configuration as follows:

> If the Canton participant to which the JSON API is connected uses target-scopes, the configuration has to reflect that
> through the following setting:

Author: ```suggestion
To connect to a Canton participant that uses target-scopes, the JSON API must also include target-scopes in its
configuration as follows:
```
